test/test_premailer.rb

```diff
@@ -302,6 +302,21 @@ def test_premailer_related_attributes
     end
   end
 
+  def test_empty_css_att
+    html = <<-END_HTML
+    <html> <head> <style>table { background-color: rgb(250, 250, 250); } </style>
```

Comment:
I would have tested the fix with an css style unrelated to background color, that was fixed with this PR.